fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/cursors/IteratorCursor.java

```diff
@@ -37,14 +39,40 @@
  */
 @API(API.Status.MAINTAINED)
 public class IteratorCursor<T> extends IteratorCursorBase<T, Iterator<T>> {
+    private static final Logger LOG = LoggerFactory.getLogger(IteratorCursor.class);
+    private Closeable closeable;
+
     public IteratorCursor(@Nonnull Executor executor, @Nonnull Iterator<T> iterator) {
         super(executor, iterator);
```

Comment:
```suggestion
        this(executor, iterator, null);
```

Just so that there's only one "real" constructor